examples/fork-join-variable-access/fork_join_variable_access.description

```diff
@@ -1,10 +1,10 @@
-// Workers within a fork-join can access the variables that are declared within the scope of that statement.
+// Workers within a `fork-join` can access the variables that are declared within the scope of that statement.
 // The variables can be value type variables or reference type variables.
 // Value type variables, such as string, integer, float, and blob type variables, are passed into the workers as a copy
 // of the original data. Therefore, you are able to update the value of it within the worker and that value is only
 // defined within the specific worker.
 // Reference type variables, such as map, json, xml, and records type variables, are passed into these workers as a
 // reference to the original data. Therefore, if you update the value of these variables within a worker, the value gets
 // updated for the entire function.
-// The Join block receives a map of any typed values, which contains the data sent from the respective workers,
+// The `Join` block receives a map of any type values, which contains the data sent from the respective workers,
```

Comment:
IMO it should be `any` typed values

Reply:
Fixed with commit https://github.com/ballerina-platform/ballerina-examples/pull/682/commits/b26a5896ea9026845360cff0f60f8680bc2b2d89